libpod/container.go

```diff
@@ -39,8 +38,12 @@ const (
 	ContainerStatePaused ContainerStatus = iota
 )
 
-// DefaultCgroupParent is the default prefix to a cgroup path in libpod
-var DefaultCgroupParent = "/libpod_parent"
+// CgroupfsDefaultCgroupParent is the cgroup parent for CGroupFS in libpod
+const CgroupfsDefaultCgroupParent = "/libpod_parent"
+
+// SystemdDefaultCgroupParent is the cgroup parent for the systemd cgroup
+// manager in libpod
+const SystemdDefaultCgroupParent = "system.slice"
```

Comment:
any particular reason for using system.slice here? my understanding from the [reading](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/resource_management_guide/sec-default_cgroup_hierarchies) i've done was that machine.slice is the preferred target for containers ("machines" being interpreted here as referring to processes launched from rootfs tarballs like OCI or machinectl supported formats, instead of normal "contained" system/user processes).

my worry with a system.slice default is namespace conflicts with system services running under system.slice. as e.g. if my pod scheduler is running etcd.service on the host, and receives an etcd pod as part of an application deployment, do they both end up under /system.slice/etcd.service? since machine.slice is unpopulated by default and intended as a home for containers, it seems like a preferable default cgroup parent, unless i'm missing something?